lms/templates/dashboard.html

```diff
@@ -194,19 +194,20 @@
                 show_courseware_link = show_courseware_links_for.get(session_id, False)
                 cert_status = cert_statuses.get(session_id)
                 can_refund_entitlement = entitlement and entitlement.is_entitlement_refundable()
+                partner_managed_enrollment = enrollment.mode == 'masters'
                 can_unenroll = (not cert_status) or cert_status.get('can_unenroll') if not unfulfilled_entitlement else False
                 credit_status = credit_statuses.get(session_id)
                 course_mode_info = all_course_modes.get(session_id)
                 is_paid_course = True if entitlement else (session_id in enrolled_courses_either_paid)
-                is_course_blocked = (session_id in block_courses)
+                is_course_blocked = (session_id in block_courses) if block_courses else (False)
```

Comment:
Not necessary to add condition for checking it (False)
If `block_courses` are empty list here then it will return (False).

Reply:
One more, I noticed, blocked_courses code is depreciated from edx-platform so I do not think we still need this line in our themes repo. 
https://github.com/edx/edx-platform/commit/6aa300711b9dc1255ecdb905883380e06c9570b6

Reply:
@asadiqbal08 regarding your concern for the condition, it's added for `undefined` value and you could see associated attached error screenshot that is a specific fix of it.

Reply:
getting `undefined` is because of it is removed from edX. Please see the ref PR.

Reply:
and regarding your concern for its deprecated I have tested it on upstream master. Yes, its deprecated but not sure about `juniper` release as I am provisioned on `upstream master`.

removing it on @asadiqbal08 suggestion.

Reply:
As per 1-1 conversation with @umarmughal824 , It seems we are on Juniper and `is_course_blocked` is not depreciated there but could not understand why @umarmughal824 are you getting `undefined` over this condition then ? can you please update me ?

Reply:
@shaidar could you let us know is that deprecated in `juniper` by verifying it on RC as both of us are provisioned on `upstream master`?

Reply:
@umarmughal824 `is_course_blocked` is still on the [xpro/juniper branch](https://github.com/mitodl/edx-platform/blob/xpro/juniper/lms/templates/dashboard.html#L203)

Reply:
then it should be good to go after approval.